packages/cli/test/dev/fixtures/runtime-not-installed/vercel.json

```diff
@@ -1,3 +1,3 @@
 {
-  "builds": [{ "src": "*.js", "use": "@vercel/does-not-exist" }]
```

Comment:
This is the only one I'm worried about. I'm thinking perhaps the `isBundledBuilder()` should still check if the name exists in the CLI package.json dependencies.